src/fluidplayer.js

```diff
@@ -1910,41 +1911,48 @@ const fluidPlayerClass = function () {
         };
         let initiateVolumebarTimerId = setInterval(initiateVolumebar, 100);
         self.destructors.push(() => clearInterval(initiateVolumebarTimerId));
-
-        if (self.displayOptions.layoutControls.doubleclickFullscreen && !(self.isTouchDevice() || self.displayOptions.layoutControls.controlForwardBackward.doubleTapMobile)) {
-            self.domRef.player.addEventListener('dblclick', self.fullscreenToggle);
+    
+        // Separate behavior for desktop and mobile
+        if (self.displayOptions.layoutControls.doubleclickFullscreen && !self.isTouchDevice()) {
+            self.domRef.player.addEventListener('dblclick', () => {
+                console.log('Double click detected for fullscreen'); // Optional for debugging
```

Comment:
We only want console logs for events like VAST ads event, or when something goes wrong. So you can remove this one.